Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,24 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Please input your First Name and press ENTER.");
```

Comment:
Almost perfect, but second person input is missing.

Reply:
Thank you. Will update these changes tonight!